### PR TITLE
former members page

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: project/former.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+=======================
+Former Members of SunPy
```
Comment (Cadair) [inline project/former.rst:2]: People on this list might be still be active in the project, so this seems a little too final. "Emeritus role holders" or "previous role holders" or something?

Comment (nabobalis) [reply project/former.rst:2]: Changed to Emeritus role holders

**Comment by ayshih** [comment]
\<ahem\> Albert Shih, Lead Developer, 2014–2015

=)

**Comment by nabobalis** [comment]
> <ahem> Albert Shih, Lead Developer, 2014–2015
> 
> =)

Woops.

File: project/former.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+   * - Name
+     - Role
+     - Period
+   * - `Andy Inglis <https://github.com/aringlis>`__
+     - Board member and Secretary
+     - 2014 - 2017
+   * - `Albert Shih <https://github.com/ayshih>`__
+     - Board member
+     - 2014 - 2018
```
Comment (nabobalis) [inline project/former.rst:19]:
```suggestion
     - Board member, Lead Developer
     - 2014 - 2018, 2014 - 2015
```

File: project/former.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
+=====================
+Emeritus role holders
+=====================
+
+This page lists the people who have served as official members of the SunPy Project in some capability.
```
Comment (Cadair) [inline project/former.rst:5]:
```suggestion
This page lists the people who have previously served in named roles in the SunPy Project.
```

**Comment by Cadair** [comment]
I don't like the wording in the menu bar either, where is that?

**Comment by nabobalis** [comment]
That is in the theme and its changed.

**Review by Cadair — APPROVED** [review]
I am happy with this with the theme change for the menubar.

Thanks a lot for making this thought of mine into a reality @nabobalis

**Comment by nabobalis** [comment]
> I am happy with this with the theme change for the menubar.
> 
> Thanks a lot for making this thought of mine into a reality @nabobalis

It's what you pay me for.

**Pull request merged**